src/soc/riscv_example_soc.c

```diff
@@ -113,6 +116,11 @@ void rv_soc_init(rv_soc_td *rv_soc, char *fw_file_name, char *dtb_file_name)
 
     write_mem_from_file(fw_file_name, soc_ram, sizeof(soc_ram));
 
+    if (initrd_file_name != NULL) {
+     printf("XXXXXXXXXXXXXXXXXXXXXXXXXXX\n");
```

Comment:
Please remove this print statement or change it to something meaningful.

Reply:
Oups... again, leftover from experimentation...